test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -3023,6 +3024,71 @@ def matcher_check_fn():
                 rtol=0.07,
             )
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    def test_linear_dynamic_fp16(self):
+        class M(torch.nn.Module):
+            def __init__(self, bias: bool):
+                super().__init__()
+                self.linear = torch.nn.Linear(256, 256, bias=bias)
+
+            def forward(self, x):
+                return self.linear(x)
+
+        quantizer = X86InductorQuantizer().set_global(
+            xiq.get_default_x86_inductor_quantization_config()
+        )
+        quantizer.set_module_type_qconfig(
+            torch.nn.Linear, xiq.get_x86_inductor_linear_dynamic_fp16_config()
+        )
+        bias_list = [True, False]
+        input_ndim_list = [2, 3]
+        x_contig_list = [True, False]
+        cases = itertools.product(bias_list, input_ndim_list, x_contig_list)
+        for bias, input_ndim, x_contig in cases:
+            x_shape = (4, 256) if input_ndim == 2 else (4, 1, 256)
+            x = torch.randn(x_shape)
+            if not x_contig:
+                x = x[0::2, ...]
```

Comment:
what is this?

Reply:
This makes `x` non-contiguous by slicing along the first dimension of `x`